src/Logic/Camera.java

```diff
@@ -1,27 +1,33 @@
+package Logic;
 
-import org.lwjgl.ovr.OVRMatrix4f;
-
-import java.io.BufferedReader;
 import java.lang.Math;
-import java.util.Vector;
 
 import org.joml.*;
 
+//used to move in 3D space
 public class Camera
 {
 	
 	public Vector3f position = new Vector3f(0,0,5.f);
+	
+	//up direction of the camera
 	public Vector3f up = new Vector3f(0,1,0);
 	
+	//the closest point that you can see
 	public float closePlane = 0.01f;
+	
+	//view distance
 	public float farPlane  = 100.f;
```

Comment:
[Fix] nu vad ce nevoie e ca acestea sa fie publice, cand sunt folosite doar in aceasta clasa

Reply:
modific farPaneul daca vreau sa vad mai departe si close planeul deobicei nu e nevoie dar pot aparea erori vizuale in cazuri extreme. De asemenea cand se implementeaza chestii mai complexe valorile astea 2 sunt citite in afara clasei. exemplu https://ogldev.org/www/tutorial49/tutorial49.html